src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -167,12 +167,18 @@ public static bool IsVisible(this MarkdownObject markdownObject)
                             visiable = visiable || VisibleHtml(line.Slice.Text);
                         }
                         return true;
-                    case HtmlInline htmlInline:
-                        visiable = visiable || VisibleHtml(htmlInline.Tag);
-                        return false;
+                    case ThematicBreakBlock _:
```

Comment:
add the new added three cases to MarkdigUtilityTest

Reply:
Heading case has been added. ThematicBreakBlock cannot generated here, it has been tested in validation spec test.